x/escrow/keeper/keeper_test.go

```
@@ -3,20 +3,94 @@ package keeper_test
 	"github.com/stretchr/testify/require"
+
 	"pkg.akt.dev/go/node/escrow/module"
 	etypes "pkg.akt.dev/go/node/escrow/types/v1"
 	"pkg.akt.dev/go/testutil"
 
-	cmocks "pkg.akt.dev/node/testutil/cosmos/mocks"
 	"pkg.akt.dev/node/testutil/state"
-	"pkg.akt.dev/node/x/escrow/keeper"
 )
 
+type kTestSuite struct {
```

golangci-lint (2.5.0)

[error] type kTestSuite is unused

(unused)